safe/utilities/help.py

```diff
@@ -39,6 +39,8 @@ def show_help(message=None):
     """
 
     help_path = mktemp('.html')
-    with open(help_path, 'w+') as f:
-        f.write(get_help_html(message))
-        QDesktopServices.openUrl(QUrl(help_path))
+    with open(help_path, 'wb+') as f:
```

Comment:
Thanks, I'll give it a try.

I don't understand why converting to bytes and writing bytes would work, but what I start to suspect is that your locale is not an UTF-8 locale.

This is the only explanation it comes to my mind.

Btw, thanks! I'll test it.

Reply:
Do you mean locale in my computer or QGIS? I use my computer locale for QGIS, and it's `en_US`

Reply:
I think that the default encoding on your testing machine is 'ascii', while on mine is 'utf8', can you try the original code and use `with open(help_path, 'w+', encoding='utf8')`?